### ISSUE TITLE
Number of metabolites in PALS table doesn't match those in side panel

### ISSUE THREAD

**Issue by kmcluskey** [kmcluskey/FlyMet#61]
Sue pointed out that at times the number of metabolites listed as being present in the PALS table does not match those that are shown in the table at the side, which is odd. 
![Screenshot (20)_LI](https://user-images.githubusercontent.com/13998965/100086365-3b73d480-2e45-11eb-8c04-400261e10ea1.jpg)

**Comment by joewandy** [comment]
how's the metabolites on the table retrieved? also from PALS? or reactome?

**Comment by kmcluskey** [comment]
From PALS - the table (I believe) is the one taken from PALS. The DS F used to be calculated separately but it should be the same as the one returned from PALS.

**Comment by joewandy** [comment]
hmmm we need a notebook that reproduces this problem. Do you think you could make it?

**Comment by kmcluskey** [comment]
Yeah - it's an odd one..I'll look into it - a PALS/FlyMet notebook will be useful (I'll have most of it somewhere but will organise it)

**Comment by kmcluskey** [comment]
This is a problem with using the related_chebi IDs from relation.tsv to identify compounds in the pathways. These related chebi ids are sent to Reactome assuming they are the same compound from which they are related (i.e. a single formula). However, some of these related chebi_ids have different compound formulas e.g. ATP is actually ATP(4-) so has 4 less H+ ions than the 'original compound. So we could either add a 'related compound formula' column to the Compound object - or give each related chebi its own compound assuming we can source all the extra formulas.
